### ISSUE TITLE
Unable to detect boss

### ISSUE THREAD

**Issue by hjk22** [Egoistically/ALAuto#94]
_Opening this as a separate issue from #79._

Sometimes, when boss spawns, the bot doesn't detect it and instead starts clearing the whole map.
It is caused by the input swipes if no ships are found.

**Comment by Macro-sudo** [comment]
I got the same issue when clearing 7-2

**Comment by nhattminhh** [comment]
A temporary fix for this is just put the "Kills before boss" higher so that all other mob ships are destroyed. This will solve the issue of "Boss icon got covered by mob ships". The obvious downside is oil cost.

**Comment by PlanetTheCloud** [comment]
Yes, and sometimes when there is no other target left (just the boss), it gets stuck and kept swiping the screen to find the boss.
